projects/bp-gallery/src/hooks/get-tags-with-thumbnail.hook.ts

```diff
@@ -7,15 +7,16 @@ import {
 } from '../graphql/APIConnector';
 import { TagType } from '../types/additionalFlatTypes';
 import useGenericTagEndpoints from './generic-endpoints.hook';
-import { WatchQueryFetchPolicy } from '@apollo/client';
+import { NUMBER_OF_PICTURES_LOADED_PER_FETCH } from './get-pictures.hook';
 
 const useGetTagsWithThumbnail = (
   queryParams: LocationTagFiltersInput | KeywordTagFiltersInput | PersonTagFiltersInput | undefined,
   thumbnailQueryParams: PictureFiltersInput | undefined,
   type: TagType,
   sortBy: string[] = ['name:asc'],
   limit: number = NUMBER_OF_PICTURES_LOADED_PER_FETCH,
-  fetchPolicy?: WatchQueryFetchPolicy
+  fetchPolicy?: WatchQueryFetchPolicy,
+  noLimit?: boolean
```

Comment:
Instead of using a new parameter in addition to the existing `limit` parameter, you could change the type of `limit` to allow for something like a symbol named `NO_LIMIT`, which also is more readable than a `true` at the end of the parameter list. For example:
```ts
export const NO_LIMIT = Symbol('NO_LIMIT');
...
  limit: number | typeof NO_LIMIT = NUMBER_OF...,
...
limit: limit === NO_LIMIT ? undefined : limit,
```